### PR TITLE
[debezium] Improve convertToBinary for RowDataDebeziumDeserializeSchema

### PR DESCRIPTION
**Pull request by gong**
dbzObj may be string when mysql cdc set with param 'debezium.binary.handling.mode' = 'base64'.
So add
```java
if (dbzObj instanceof String) {
     String data = (String) dbzObj;
     return data.getBytes(StandardCharsets.UTF_8);
  } 
```

### CODE REVIEW

**Comment by gong** [comment]
@ruanhang1993 Hello, please help review it.

**Comment by gong** [comment]
The pipeline is `TiDBE2eITCase` exception. It is not relative with this PR.

**Comment by gong** [comment]
@leonardBang PTAL

**Review by leonardBang — COMMENTED** [review]
Thanks @gong for the contribution, the change looks good to me, could you add a test for this change?

**Comment by gong** [comment]
> Thanks @gong for the contribution, the change looks good to me, could you add a test for this change?

ok

**Comment by gong** [comment]
> > Thanks @gong for the contribution, the change looks good to me, could you add a test for this change?
@leonardBang ok. I will add UT.

**Comment by gong** [comment]
@leonardBang I add an UT.

File: flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java [hunk 1]
```diff
@@ -2269,4 +2269,87 @@ private static void waitForSnapshotStarted(CloseableIterator<Row> iterator) thro
...
+    public void testBinaryHandlingModeWithBase64() throws Exception {
+        if (!incrementalSnapshot) {
+            return;
+        }
+        inventoryDatabase.createAndInitialize();
+        String sourceDDL =
+                String.format(
+                        "CREATE TABLE varbinary_base64_table ("
+                                + " id INT,"
+                                + " order_id STRING,"
```
Comment (leonardBang) [inline flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MySqlConnectorITCase.java:2283]: MySQL `varbinary` type should map to Flink SQL `bytes`  ？

**Comment by gong** [comment]
> Thanks @gong for the explanation and ITCase, the code change part looks good to me.
> 
> Could you add parameter 'debezium.binary.handling.mode' = 'base64' to cdc connector option part of docs ? A short paragraph to describe the user case and parameter is recommended

@leonardBang Ok. I will add docs to decribe the user case

**Review by leonardBang — APPROVED** [review]
Thanks @gong for the update, LGTM . Will merge this PR once the CI green

**Comment by leonardBang** [comment]
The failed test is not related to current PR, merging.

**Pull request merged**